controllers/operator/configmap.go

```diff
@@ -966,6 +976,18 @@ func getClusterAddress(client client.Client, namespace string, configMap string)
 	return "", errors.New(fmt.Sprint("failed to fetch the cluster address"))
 }
 
+func CheckSecretExists(client client.Client, namespace string, auditSecretName string) (bool, error) {
```

Comment:
Probably rename this to `checkSecretExists` given we don't need to expose this function outside of this package.